src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { CaseFlagFieldState, SelectFlagLocationErrorMessage } from '../../enums';
+import { ManageCaseFlagsComponent } from './manage-case-flags.component';
+
+describe('ManageCaseFlagsComponent', () => {
+  let component: ManageCaseFlagsComponent;
+  let fixture: ComponentFixture<ManageCaseFlagsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [ ReactiveFormsModule ],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+      declarations: [ManageCaseFlagsComponent ]
```

Comment:
@tarikdogan-hmcts - missing space between `[` and `M` (for consistency of formatting)